monitoring/configuring-the-monitoring-stack.adoc

```diff
@@ -108,13 +108,16 @@ include::modules/monitoring-modifying-retention-time-for-prometheus-metrics-data
 
 // Configuring remote write storage for Prometheus
 include::modules/monitoring-configuring-remote-write-storage.adoc[leveloffset=+1]
+// Configuring remote write authentication methods for Prometheus
+include::modules/monitoring-supported-remote-write-authentication-settings.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 .Additional resources
 
 * See link:https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage[Setting up remote write compatible endpoints] for steps to create a remote write compatible endpoint (such as Thanos).
 * See link:https://prometheus.io/docs/practices/remote_write/#remote-write-tuning[Tuning remote write settings] for information about how to optimize remote write settings for different use cases.
-* For information about additional optional fields, please refer to the API documentation.
+* See xref:../nodes/pods/nodes-pods-secrets.adoc#nodes-pods-secrets-about_nodes-pods-secrets[Understanding secrets] for steps to create and configure `Secret` objects in {product-title}.
+* For information about additional optional fields, see the xref:../rest_api/monitoring_apis/prometheus-monitoring-coreos-com-v1.adoc[Prometheus REST API reference].
```

Comment:
Here, WDYT about linking instead the API Reference only for the Spec.RemoteWrite part? [1]

[1] https://deploy-preview-44713--osdocs.netlify.app/openshift-enterprise/latest/rest_api/monitoring_apis/prometheus-monitoring-coreos-com-v1.html#spec-remotewrite-2

Reply:
Yes. That makes sense. I'll change it.